druid/src/widget/svg.rs

```diff
@@ -85,12 +86,22 @@ impl<T: Data> Widget<T> for Svg<T> {
     }
     fn paint(&mut self, paint_ctx: &mut PaintCtx, _data: &T, _env: &Env) {
         //TODO: options for aspect ratio or scaling based on height
-        let scale = paint_ctx.size().width / self.get_size().width;
+        let scalex = paint_ctx.size().width / self.get_size().width;
+        let scaley = paint_ctx.size().height / self.get_size().height;
+        let scale = scalex.min(scaley);
 
         let origin_x = (paint_ctx.size().width - (self.get_size().width * scale)) / 2.0;
         let origin_y = (paint_ctx.size().height - (self.get_size().height * scale)) / 2.0;
         let origin = Point::new(origin_x, origin_y);
 
+        let clip_rect = Rect::from_origin_size(
+            Point::ORIGIN,
+            Size::new(paint_ctx.size().width, paint_ctx.size().height),
```

Comment:
This can just be `paint_ctx.size()` directly, that returns a `Size` struct.

A sort-of-sneaky way I sometimes would write this is,

```rust
let clip_rect = Rect::ZERO.with_size(paint_ctx.size());
```

`with_size` is a method on `Rect` that returns a new `Rect` with the same origin as `self`, but with a new size. 🤷‍♂

Reply:
Applied, thanks